### DISCUSSION TITLE
osu!lazer crashes

### DISCUSSION THREAD

**Discussion by 100925afg** [ppy/osu#28046] [Q&A] [answered]
https://github.com/ppy/osu/assets/119842900/82376cb3-9241-4130-b1d7-87fbf3b21e4b

Last night I switched the rendering engine from d3d11 to vulkan ,then it crashed.I've tried to reinstall osu!lazer but it didn't work.How can I switch to d3d11? I have searched issues about vulkan:
https://github.com/ppy/osu/issues/21314
https://github.com/ppy/osu/issues/12084
https://github.com/ppy/osu/issues/4593

[logs.zip](https://github.com/ppy/osu/files/15172956/logs.zip)
I upload the video and the logs.
OS: Windows10 Professional Version
GPU: Nvidia GeForce GT710
CPU: Intel(R) Xeon(R) CPU           X5670  @ 2.93GHz   2.93 GHz
Thanks to anyone who can help me.

**Comment by 100925afg** [comment]
> 2024-05-01.10-32-39.mp4 
> 昨晚我将渲染引擎从 d3d11 切换到 vulkan，然后它崩溃了。我试图重新安装osu！lazer，但它没有用。如何切换到 d3d11？我已经搜索了有关 vulkan 的问题：#21314、#12084、#4593
> 
> [logs.zip](https://github.com/ppy/osu/files/15172956/logs.zip)我上传视频和日志。操作系统： Windows10 Professional Version GPU： Nvidia GeForce GT710 CPU： Intel（R） Xeon（R） CPU X5670 @ 2.93GHz 2.93 GHz 感谢任何可以帮助我的人。

when i use the keyboard or mouse in this osu,it have reaction,but still lag.After I press the confirm button of the prompt box,the osu will close automatically。

**Comment by BlockHead007** [comment, marked as answer]
Vulkan on lazer is pretty unstable and can cause crashes and memory leaks (see #27659). The only recommendation is to switch back to d3d11 or ogl through games options or if you can't reach it go to games data location (default at %appdata%\osu ), open framework.ini and set Renderer = Automatic.

**Reply by 100925afg** [reply]
Thank you!That's useful for me.
